### PR TITLE
[Breaking change] Fix mouse wheel value on win32 and return a f64 instead of i32 in the MouseWheel event

### LINKED ISSUE #260: 2D mouse wheel support
Currently it supports only vertical scrolling.

GLFW and SDL2 implements this as a relative 2D coordinate.

### PR DESCRIPTION
**Pull request by tomaka**
Close #260

### CODE REVIEW

**Comment by retep998** [comment]
:cactus:

**Comment by fkaa** [comment]
I think while we're at it we might as well add horizontal scrolling (`MouseWheel(f64, f64)`), which will close #260.

**Comment by tomaka** [comment]
Oh, right.

File: src/api/cocoa/mod.rs [hunk 1]
```diff
@@ -275,7 +275,7 @@ impl<'a> Iterator for PollEventsIterator<'a> {
                     self.window.delegate.state.pending_events.lock().unwrap().extend(events.into_iter());
                     event
                 },
-                NSScrollWheel           => { Some(MouseWheel(event.scrollingDeltaY() as i32)) },
+                NSScrollWheel           => { Some(MouseWheel(event.scrollingDeltaX() as f64, event.scrollingDeltaY() as f64)) },
```
Comment (tomaka) [inline src/api/cocoa/mod.rs]: So I guess that `event.scrollingDeltaX()` probably exists, even though I didn't check.

Comment (fkaa) [inline src/api/cocoa/mod.rs]: Correct!

**Comment by tomaka** [comment]
Will merge before releasing 0.1.

**Pull request merged**